editor/src/core/shared/file-utils.ts

```diff
@@ -259,3 +259,8 @@ export function isCssFile(filename: string): boolean {
 export function isJsOrTsFile(filename: string): boolean {
   return isJsFile(filename) || isTsFile(filename)
 }
+
+export function isImageUrl(url: string): boolean {
```

Comment:
there's also `isImage` function, we might be able to unify them
https://github.com/concrete-utopia/utopia/blob/3f73ea881b22126e3fe104d04ef19219c5588ff9/editor/src/core/shared/utils.ts#L212-L214

Reply:
nice catch!